app/_src/gateway-operator/reference/version-compatibility.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Version Compatibility
+---
+
+## {{ site.kic_product_name }}
+
+The following table presents the general compatibility of {{site.kgo_product_name}} with {{ site.kic_product_name }} minor versions.
+
+## Kubernetes
+
+| {{site.kgo_product_name}}        |            1.0.x            |
+|:---------------------------------|:---------------------------:|
+| {{ site.kic_product_name }} 2.11 | <i class="fa fa-check"></i> |
+| {{ site.kic_product_name }} 2.12 | <i class="fa fa-check"></i> |
+
+### General
+
+The following table presents the general compatibility of {{site.kgo_product_name}} with specific Kubernetes versions.
+Users should expect all the combinations marked with <i class="fa fa-check"></i> to work and to be supported.
+
+| {{site.kgo_product_name}} |            1.0.x            |
+|:--------------------------|:---------------------------:|
+| Kubernetes 1.25           | <i class="fa fa-check"></i> |
+| Kubernetes 1.26           | <i class="fa fa-check"></i> |
+| Kubernetes 1.27           | <i class="fa fa-check"></i> |
+| Kubernetes 1.28           | <i class="fa fa-check"></i> |
+
+### Gateway API
+
+The following table presents the compatibility of {{site.kgo_product_name}} with specific [Gateway API][gateway-api] versions.
+As {{site.kgo_product_name}} implements Gateway API features using the upstream
+project, which defines [its own compatibility declarations][gateway-api-supported-versions], the expected compatibility
+of Gateway API features might be limited to those.
+
+| {{site.kgo_product_name}} |            1.0.x            |
+|:--------------------------|:---------------------------:|
+| Gateway API 0.8.1         | <i class="fa fa-check"></i> |
```

Comment:
```suggestion
| Gateway API 1.0.0         | <i class="fa fa-times"></i> |
| Gateway API 0.8.1         | <i class="fa fa-check"></i> |
```

Reply:
I wasn't sure about this one so I needed to check it and it seems that we should be able to say we support v1 Gateway API with KGO v1.0.x.

That's because `v1beta1` is still served **and set as storage version** in v1 https://github.com/kubernetes-sigs/gateway-api/blob/v1.0.0/config/crd/standard/gateway.networking.k8s.io_gateways.yaml#L1710.

We don't test this in our CI (install either 0.8 or 1.0 of Gateway API and then run the operator). We only use one version of Gateway API for all tests. So while my local test proves it might work it might not for all the objects and use cases.

OTOH we might not mark it support GW API v1 and use KGO v1.1 as an opportunity to bump users to a newer version.